server/db/seeds/foods.js

```diff
@@ -3,32 +3,32 @@ exports.seed = (knex) =>
-        { id: 8, name: 'Carrots', reo_name: 'Kāroti', average_price: 2.10 },
-        { id: 9, name: 'Cauliflower', reo_name: 'Kareparāoa', average_price: 3.36 },
-        { id: 10, name: 'Celery', reo_name: 'Herewī', average_price: 3.31 },
-        { id: 11, name: 'Courgette', reo_name: 'Roroa iti', average_price: 8.34 },
-        { id: 12, name: 'Cucumber', reo_name: 'Kūkama', average_price: 7.43 },
-        { id: 13, name: 'Grapes', reo_name: 'Kerepi', average_price: 7.30 },
-        { id: 14, name: 'Kiwifruit', reo_name: 'Huakiwi', average_price: 3.58 },
-        { id: 15, reo_name: 'Kūmara', average_price: 5.29 },
-        { id: 16, name: 'Lettuce', reo_name: 'Rētihi', average_price: 4.30 },
-        { id: 17, name: 'Mandarins', reo_name: 'Manarini', average_price: 5.22 },
-        { id: 18, name: 'Mushrooms', reo_name: 'Harore', average_price: 10.57 },
-        { id: 19, name: 'Onions', reo_name: 'Riki', average_price: 2.07 },
-        { id: 20, name: 'Oranges', reo_name: 'Ārani', average_price: 3.29 },
-        { id: 21, name: 'Parsnips', reo_name: 'Tāmore mā', average_price: 3.70 },
-        { id: 22, name: 'Pears', reo_name: 'Pea', average_price: 3.70 },
-        { id: 23, name: 'Pineapple', reo_name: 'Paināporo', average_price: 3.29 },
-        { id: 24, name: 'Potatoes', reo_name: 'Rīwai', average_price: 1.69 },
-        { id: 25, name: 'Pumpkin', reo_name: 'Paukena', average_price: 2.53 },
-        { id: 26, name: 'Tomatoes', reo_name: 'Tōmato', average_price: 6.03 }
+        { id: 1, name: 'Apples', reo_name: 'Āporo', average_price: 2.80, nz_grown: true },
+        { id: 2, name: 'Avocados', reo_name: 'Awakato', average_price: 9.71, nz_grown: true },
+        { id: 3, name: 'Bananas', reo_name: 'Panana', average_price: 2.71, nz_grown: false },
+        { id: 4, name: 'Beans', reo_name: 'Pīni', average_price: 12.61, nz_grown: true },
+        { id: 5, name: 'Broccoli', reo_name: 'Porokori', average_price: 5.76, nz_grown: true },
+        { id: 6, name: 'Cabbage', reo_name: 'Kāpeti', average_price: 1.94, nz_grown: true },
+        { id: 7, name: 'Capsicum', reo_name: 'Rapikama', average_price: 12.33, nz_grown: true },
+        { id: 8, name: 'Carrots', reo_name: 'Kāroti', average_price: 2.10, nz_grown: true },
+        { id: 9, name: 'Cauliflower', reo_name: 'Kareparāoa', average_price: 3.36, nz_grown: true },
+        { id: 10, name: 'Celery', reo_name: 'Herewī', average_price: 3.31, nz_grown: true },
+        { id: 11, name: 'Courgette', reo_name: 'Roroa iti', average_price: 8.34, nz_grown: true },
+        { id: 12, name: 'Cucumber', reo_name: 'Kūkama', average_price: 7.43, nz_grown: true },
+        { id: 13, name: 'Grapes', reo_name: 'Kerepi', average_price: 7.30, nz_grown: false },
+        { id: 14, name: 'Kiwifruit', reo_name: 'Huakiwi', average_price: 3.58, nz_grown: true },
+        { id: 15, name: 'Kumara', reo_name: 'Kūmara', average_price: 5.29, nz_grown: true },
```

Comment:
the name for kumara shouldn't be here 
```suggestion
        { id: 15,  reo_name: 'Kūmara', average_price: 5.29, nz_grown: true },
```